### PR TITLE
Frontendstuff

### PR DESCRIPTION
**Pull request by Daniel-Redfern**
Adds review display functionality.

### CODE REVIEW

File: frontend/src/index.css [hunk 1]
```diff
@@ -582,6 +582,13 @@ dd {
     padding: 30px 0;
 }
 
+.scrollable-div {
+  max-height: 400px;
+  overflow: auto;
+  padding: 20px;
+}
+
```
Comment (jnlao) [inline frontend/src/index.css:590]: you can delete this extra blank line but everything else looks good!

**Review by jnlao — CHANGES_REQUESTED** [review]

File: frontend/src/Reviews.js [hunk 2]
```diff
@@ -1,57 +1,106 @@
...
+
+    const handleSubmit = async (event) => {
+        if(rFlag === "True"){
+            alert("Cannot Add Another Review");
+            event.preventDefault();
+        }
+        else {
+            const review = document.getElementById("reviewInputField").value;
+            setRFlag("True");
+            event.preventDefault();
```
Comment (AlexBuchko) [inline frontend/src/Reviews.js:88]: event.preventDefault() could be moved to the top of the function so it doesn't need to be stated in each branch

File: frontend/src/Reviews.js [hunk 3]
```diff
@@ -74,9 +123,14 @@ export default function Reviews(props) {
...
     };
+
     return (
         <div>
-            <h2>Reviews</h2>
+            <h3>Written Reviews</h3>
+            <div className="scrollable-div">
+                <ReviewDisplay reviews={reviews}></ReviewDisplay>
+            </div>
+            <h3>Numerical Reviews</h3>
```
Comment (AlexBuchko) [inline frontend/src/Reviews.js:133]: Could this be changed to "Ratings" instead of "Numerical Reviews"?

**Review by AlexBuchko — CHANGES_REQUESTED** [review]

**Comment by AlexBuchko** [comment]
Should probably fix whatever errors / warnings are causing the build to fail

File: frontend/src/Reviews.js [hunk 4]
```diff
@@ -1,57 +1,105 @@
...
+            }
+        });
+    }, []);
 
     const handleClick = async (type) => {
-        try {
-            const response = await axios.put(
-                `http://localhost:4000/foods/${food._id}/${type}`
-            );
+        if((type === "likes" && lFlag === "True") || (type === "dislikes" && dFlag === "True") || (type === "poisonings" && pFlag === "True")){
```
Comment (Ximixiyou) [inline frontend/src/Reviews.js:27]: this line is a bit confusing, could you leave a comment on it?

**Review by Ximixiyou — COMMENTED** [review]
everything looks good, good job

**Pull request merged**